helm/hpcc/templates/_helpers.tpl

```diff
@@ -1045,11 +1045,42 @@ Pass in dict with root, category.  optional name to restrict it to a single name
 
 {{/*
 Create placement related settings
-Pass in dict with placement
+Pass in dict with current placment and new placements dictinary for this entity
+*/}}
+{{- define "hpcc.addPlacementSetting" -}}
```

Comment:
minor: mergePlacementSettings slightly more accurate

Reply:
Should have a function comment something like 'Pass in dict with me and new'

Reply:
fixed